skymp5-server/cpp/server_guest_lib/PartOne.cpp

```diff
@@ -548,15 +548,25 @@ void PartOne::Init()
     uint32_t worldOrCell =
       emitter->GetCellOrWorld().ToFormId(worldState.espmFiles);
 
+    // See 'perf: improve game framerate #1186'
+    // Client needs to know if it is DOOR or not
+    const char* baseRecordTypePrefix = "";
+    std::string baseRecordType;
+    if (const std::string& baseType = emitter->GetBaseType();
+        baseType == "DOOR") {
+      baseRecordTypePrefix = R"(, "baseRecordType": )";
+      baseRecordType = '"' + baseType + '"';
+    }
+
     Networking::SendFormatted(
       sendTarget, listenerUserId,
       R"({"type": "%s", "idx": %u, "isMe": %s, "transform": {"pos":
-    [%f,%f,%f], "rot": [%f,%f,%f], "worldOrCell": %u}%s%s%s%s%s%s%s%s%s%s%s})",
+    [%f,%f,%f], "rot": [%f,%f,%f], "worldOrCell": %u}%s%s%s%s%s%s%s%s%s%s%s%s%s})",
```

Comment:
And what would you think about just using a JSON library here instead? I don't think that it would make a huge performance difference, but it'd surely make a much cleaner code

(Not an actual request to change anything)

Reply:
nlohmann::json is much slower

Reply:
That much?

Reply:
You can test this on 1M cycles. nlohman::json is quite slow library.

Reply:
Not sure if it's THAT slow that it would lead to us having some performance problems (given the current CPU load caused by the server isn't high). But ok